### PR TITLE
add(main): new package websocat

### LINKED ISSUE #14929: [Package]: websocat
### Why do you want to add this package to Termux?

I use this package to test websocket connections to my homelab on most of my other systems. This package serves the same purpose as socat but works with websockets connections.

### Home page URL

https://github.com/vi/websocat

### Source code URL

https://github.com/vi/websocat

### Packaging policy acknowledgement

- [X] The requested package complies with [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy).

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by 2096779623**
Close #14929 .

### CODE REVIEW

File: packages/websocat/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/vi/websocat
+TERMUX_PKG_DESCRIPTION="Command-line client for WebSockets, like netcat (or curl) for ws:// with advanced socat-like functions"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.11.0
+TERMUX_PKG_SRCURL=https://github.com/vi/websocat/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=943d6f66045658cca7341dd89fe1c2f5bdac62f4a3c7be40251b810bc811794f
+TERMUX_PKG_BUILD_DEPENDS="openssl"
```
Comment (xtkoba) [inline packages/websocat/build.sh:8]: `openssl` is a runtime dependency.

Comment (2096779623) [reply packages/websocat/build.sh:8]: https://github.com/2096779623/termux-packages/actions/runs/4090015747/jobs/7053134230

> # openssl-sys supports OpenSSL 3 in >= 0.9.69

Comment (xtkoba) [reply packages/websocat/build.sh:8]: Did you check the artifacts:

```bash
readelf -d $PREFIX/bin/websocat | grep NEEDED
```

Comment (xtkoba) [reply packages/websocat/build.sh:8]: (You know what I mean?)

Comment (2096779623) [reply packages/websocat/build.sh]: > (You know what I mean?)

I may have misunderstood you before.

**Comment by weikinhuang** [comment]
thanks!

**Pull request merged**